src/mediapipe/imageSegmenter/audioNativeRenderer/MatrixAudioRenderer.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE in the repository root for full details.
+*/
+
+import { getTrackReferenceId } from "@livekit/components-core";
+import { type RemoteAudioTrack, Track } from "livekit-client";
+import { useEffect, useMemo, useRef, type ReactNode } from "react";
+import { useTracks, AudioTrack } from "@livekit/components-react";
+import { type CallMembership } from "matrix-js-sdk/lib/matrixrtc";
+import { logger } from "matrix-js-sdk/lib/logger";
+
+export interface CopiedAudioData {
+  data: number[];
+  sampleRate: number;
+  channels: number;
+  timestamp: number;
+}
+
+/** @public */
+export interface RoomAudioRendererProps {
+  /** Does an additional gain control on all audio tracks rendered by this component. This does not interfere with the
+   * `setVolume` of the track itself.
+   */
+  volumeFactor?: number;
```

Comment:
Bit confused by this. Is the volumeFactor cumulative or just the default?

I.e. is it

1. `volumeFactor * trackVolumeFactor` for each track
2. `trackVolumeFactor ||  volumeFactor` for each track?